code/Gamemodes/Modes/FFA/FreeForAll.cs

```diff
@@ -91,6 +93,12 @@ private async Task NextTurn()
 
 		ActivePlayer.EndTurn();
 
+		if ( PlayerTurnQueue.Count == 0 )
+		{
+			RoundsUntilSuddenDeath -= 1;

```

Comment:
Maybe we should move this inside of 

```
		if ( SuddenDeath )
			await Terrain.LowerTerrain( GrubsConfig.SuddenDeathAggression );
```

?

Reply:
This makes sense to me, but PlayerTurnQueue is only in FreeForAll.cs. I think Gamemode.cs should get a bit of a refactor when we write more gamemodes, but should I move PlayerTurnQueue up + implement this change for now?